packages/input_schema/src/utilities.ts

```diff
@@ -247,6 +247,50 @@ export function validateInputUsingValidator(
                 }
             }
         }
+
+        // Check datepicker editor format
+        if (type === 'string' && editor === 'datepicker' && value && typeof value === 'string') {
+            const acceptAbsolute = isAbsolute !== false;
+            const acceptRelative = isRelative === true;
+            const isValidAbsolute = /^[0-9]{4}-[0-9]{2}-[0-9]{2}$/.test(value);
+            const isValidRelative = /^[+-] [0-9]+ (day|week|month|year)s?$/.test(value);
```

Comment:
The docs do not specify the unit. What about minutes and seconds?

Reply:
I got it, it is date picker, so the minutes and seconds are not valid, so please add this to docs, and allow units. I will add note in doc PR as well.

Reply:
Added to docs PR